mkdocs.yml

```diff
@@ -50,6 +50,9 @@ nav:
   - 'Permissions':
     - 'Guide': permissions.md
     - 'API': api/permissions/
+  - 'Navigation':
```

Comment:
Seeing that we'll have navigation-compose-material as well, how do you feel about changing this module to be navigation-animation so that we can keep the dependencies separated? Or are we good with having it all in one module? :)

Reply:
+1. I think this module should be `navigation-animation`.

> Seeing that we'll have navigation-compose-material as well...

Will we?

Reply:
changed to `navigation-animation`

Reply:
The intent is for `compose-material` dependent APIs to be separate from the core `navigation-compose` library (as not every Compose app will be a Compose Material app), so that would indeed imply a future `navigation-compose-material` artifact. We're not there yet though :smile: